server/controllers/services/email.js

```diff
@@ -0,0 +1,175 @@
+    const list = tokens[1];
+    const slug = tokens[2];
+    email.from = email.from || `${slug} collective <info@${slug}.opencollective.com>`;
+
+    return models.Notification.findAll(
+      { 
+        where: { 
+          channel: 'email',
+          type: `mailinglist.${list}`
+        },
+        include: [{model: models.User }, {model: models.Group, where: { slug } }] 
+      }
+    )
+    .tap(users => {
+      if (users.length === 0) throw new errors.NotFound(`There is no user subscribed to ${email.recipient}`);
+    })
+    .then(results => results.map(r => r.User.email))
+    .then(recipients => {
+      console.log(`Sending email from ${email.from} to ${to} (${recipients.length} recipients)`);
+      if (email.template) {
+        console.log(`preview: http://localhost:3060/templates/email/${email.template}?data=${encodeURIComponent(JSON.stringify(email))}`);
+        return emailLib.send(email.template, to, email, { from: email.from, bcc: recipients.join(',') });
+      } else {
+        console.log("Subject: ", email.subject);
+        email.body += '\n<!-- OpenCollective.com -->\n'; // watermark to identify if email has already been processed
+        return emailLib.sendMessage(to, email.subject, email.body, { from: email.from, bcc: recipients.join(',') });
+      }
+    })
+    .catch(e => {
+      console.error("error in sendEmailToList", e);
+    });
+  };
+
+  const approve = (req, res, next) => {
+      const messageId = req.query.messageId;
+      const approverEmail = req.query.approver;
+
+      let approver, sender;
+      let email = {};
+
+      const fetchSenderAndApprover = (email) => {
+        const where = { '$or': [ {email: approverEmail}, { email: email.sender } ] };
+        sender = { name: email.From, email: email.sender }; // default value
+        return models.User.findAll({ where })
+                .then(users => { 
+                  users.map(user => {
+                    if (approverEmail == user.email) approver = user; 
+                    if (email.sender == user.email) sender = user;
+                  })
+                })
+                .catch(e => {
+                  console.error("err: ", e);
+                });
+      };
+
+      const requestOptions = {
+        json: true,
+        auth: {
+          user: 'api',
+          pass: config.mailgun.api_key
+        }
+      };
+
+      request
+      .get(`https://so.api.mailgun.net/v3/domains/opencollective.com/messages/${messageId}`, requestOptions)
+      .then(json => {
+        email = json; return email;
+      })
+      .then(fetchSenderAndApprover)
+      .then(() => {
+        const emailData = {
+          template: 'email.message',
+          subject: email.Subject,
+          body: email['body-html'] || email['body-plain'],
+          to: email.To,
+          sender: _.pick(sender, ['email', 'name', 'avatar'])
+        }
+        if ( approver && approver.email != sender.email )
```

Comment:
same goes for `!=`. http://stackoverflow.com/questions/359494/which-equals-operator-vs-should-be-used-in-javascript-comparisons